src/settings/components/RoleDetails/RoleDetails.js

```diff
@@ -42,7 +41,7 @@ const RoleDetails = ({ onClose, roleId }) => {
 
   const getActionMenu = () => (
     <>
-      <Button buttonStyle="dropdownItem" onClick={() => history.push(`${pathname}?layout=edit&id=${roleId}`)}>
+      <Button buttonStyle="dropdownItem" onClick={() => history.push(`/settings/authorization-roles/${roleId}?layout=edit`)}>
```

Comment:
If somebody updates `package.json` to change [stripes.route](https://github.com/folio-org/ui-authorization-roles/blob/bd1c327a24666faa0d6d6a79e0ef8362bc2a3c78/package.json#L25) to `authz-roles`, this will break. 

Please explain why this change was necessary. It feels like a step in the wrong direction.